src/utils/settings.cpp

```diff
@@ -85,9 +85,9 @@ void check_status(const QSettings& settings, const QString& attempted_operation)
 {
     auto status = settings.status();
     if (status || exists_but_unreadable(settings.fileName()))
-        throw mp::PersistentSettingsException{attempted_operation, status == QSettings::FormatError
-                                                                       ? QStringLiteral("format")
-                                                                       : QStringLiteral("access")};
+        throw mp::PersistentSettingsException{
+            attempted_operation, status == QSettings::FormatError ? QStringLiteral("format error")
+                                                                  : QStringLiteral("access error (consider `sudo`)")};
```

Comment:
How will we approach this for Windows, where `sudo` is not a thing?

Reply:
Good point. How about "consider running this with an administrative role"? (Replacing it here and having the same msg in all platforms)

Reply:
Works for me.